subprojects/core/src/integTest/groovy/org/gradle/api/tasks/CacheTaskArchiveErrorIntegrationTest.groovy

```diff
@@ -117,7 +121,7 @@ class CacheTaskArchiveErrorIntegrationTest extends AbstractIntegrationSpec {
 
         then:
         fails("clean", "customTask")
-        failureHasCause("Failed to load cache entry for task ':customTask'")
+        failureHasCause(~/Failed to load cache entry $CACHE_KEY_PATTERN for task ':customTask': Remote build cache load exception: Not in GZIP format/)
```

Comment:
Exception messages should describe what happened that caused the exception. "Remote build cache load exception" describes the type of exception instead. I think something like this would be better:

```suggestion
        failureHasCause(~/Failed to load cache entry $CACHE_KEY_PATTERN for task ':customTask': Could not load from remote cache: Not in GZIP format/)
```

Reply:
fixed